functions/Find-DbaDbUnusedIndex.ps1

```diff
@@ -35,6 +35,30 @@ function Find-DbaDbUnusedIndex {
         This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
         Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
 
+    .PARAMETER UserSeeksLessThan
+        Specify a custom threshold for user seeks. The default for this parameter is 1.
+        The valid values are between 1 and 1000000 to provide flexibility on the definition of 'unused' indexes.
+        Note: The resulting WHERE clause uses the AND operator:
+        user_seeks < $UserSeeksLessThan
+        AND user_scans < $UserScansLessThan
+        AND user_lookups < $UserLookupsLessThan
+
+    .PARAMETER UserScansLessThan
+        Specify a custom threshold for user scans. The default for this parameter is 1.
+        The valid values are between 1 and 1000000 to provide flexibility on the definition of 'unused' indexes.
+        Note: The resulting WHERE clause uses the AND operator:
+        user_seeks < $UserSeeksLessThan
+        AND user_scans < $UserScansLessThan
+        AND user_lookups < $UserLookupsLessThan
+
+    .PARAMETER UserLookupsLessThan
```

Comment:
If we are going to assume these will be the "max limit" and the comparison will be less than said value...then we can drop the `LessThan` from the parameter naming and simply document it.

So we can just keep them as `UserSeeks` `UserScans` `UserLookups`, or really we can just do `Seeks` `Scans` and `Lookups`.  It is well known they are user based stats, so do we really need the "user" in the parameter name, @potatoqualitee ?

Reply:
I can change the param names to Seeks, Scans, and Lookups. @potatoqualitee, do you have a strong opinion either way on the names for these command line params?